### PR TITLE
Fixed the legal statement

### LINKED ISSUE #1: Legal statement
Currently, the legal statement says this, which is not exactly correct:

> UAVCAN is an interface standard open to everyone. No copyrights are reserved and no licensing is necessary for its implementation, distribution, or use. In no event shall the authors of the standard be liable for any damage arising, directly or indirectly, from its use.

A better wording should be found. The objectives are the following:

* We want the spec and related docs to be free and open in the spirit of [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) except that the "No additional restrictions" limitation should be lifted.
* The liability disclaimer should be there.

### PR DESCRIPTION
**Pull request by pavel-kirienko**
Using CC BY 4.0 as advised.

Fixes #1.

image

### CODE REVIEW

**Comment by thirtytwobits** [comment]
Looks good to me.

**Comment by thirtytwobits** [comment]
Actually, we're missing a "no warranties" statement.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by thirtytwobits** [comment]
Xmpp's statement is more comprehensive:

Disclaimer of Warranty

## NOTE WELL: This Specification is provided on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied, including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE. ##
Limitation of Liability

In no event and under no legal theory, whether in tort (including negligence), contract, or otherwise, unless required by applicable law (such as deliberate and grossly negligent acts) or agreed to in writing, shall the XMPP Standards Foundation or any author of this Specification be liable for damages, including any direct, indirect, special, incidental, or consequential damages of any character arising from, out of, or in connection with the Specification or the implementation, deployment, or other use of the Specification (including but not limited to damages for loss of goodwill, work stoppage, computer failure or malfunction, or any and all other commercial damages or losses), even if the XMPP Standards Foundation or such author has been advised of the possibility of such damages.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pavel-kirienko** [comment]
Added two more sections right after the title page: "Disclaimer of warranty" and "Limitation of liability". See the attached doc. 
[UAVCAN_Specification.pdf](https://github.com/UAVCAN/specification/files/2366116/UAVCAN_Specification.pdf)

The latter makes two references to "UAVCAN Development Team", although this entity is not explicitly defined anywhere. I hope that is acceptable.

**Comment by thirtytwobits** [comment]
How about we just omit the references?

```
...or agreed to in writing, shall any
author of this Specification be liable... 
even if such author has been advised
of the possibility of such damages.
```

_(sheesh, Lawyers!)_

**Comment by pavel-kirienko** [comment]
Done.

File: specification/UAVCAN_Specification.tex [hunk 1]
```diff
@@ -51,14 +51,41 @@ \section*{Support and feedback}
...
+\section*{Limitation of liability}
 
-This UAVCAN specification is distributed under the terms of the Creative Commons Attribution 4.0 International License.
+In no event and under no legal theory, whether in tort (including negligence), contract, or otherwise,
+unless required by applicable law (such as deliberate and grossly negligent acts) or agreed to in writing,
+shall any author of this Specification be liable for damages,
+including any direct, indirect, special, incidental, or consequential damages of any character arising
+from, out of, or in connection with the Specification or the implementation, deployment,
+or other use of the Specification (including but not limited to damages for loss of goodwill,
+work stoppage, computer failure or malfunction, or any and all other commercial damages or losses),
```
Comment (thirtytwobits) [inline specification/UAVCAN_Specification.tex]: > (including but not limited to damages for loss of goodwill,
work stoppage, computer failure or malfunction, or any and all other commercial damages or losses)

Perhaps we should make this example more domain specific. My layman's guess is that in litigation such examples serve to strengthen the "we couldn't have been more clear" argument. So how about:

(including but not limited to damages for loss of goodwill,
work stoppage, equipment failure or malfunction, injuries to persons, or any and all other commercial damages or losses)

**Comment by pavel-kirienko** [comment]
Done.

**Pull request merged**